modules/oadp-deleting-cluster-resources-following-success.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-deleting-cluster-resources-following-success_{context}"]
+= Deleting cluster resources following a successful backup and restore using Data Mover
+
+You can delete any `volumeSnapshotBackup/volumeSnapshotRestore` CRs that remain in your application namespace after a successful backup and restore in which you used Data Mover.
```

Comment:
* Should avoid using a slash
* Usually CRs start with a capital, should these be capital V's?


```suggestion
You can delete any `VolumeSnapshotBackup` or `VolumeSnapshotRestore` CRs that remain in your application namespace after a successful backup and restore in which you used Data Mover.
```

Reply:
OK, that was just ... dumb